docs/azure_arc_jumpstart/azure_arc_servers/oci/oci_terraform_ubuntu/_index.md

```diff
@@ -0,0 +1,198 @@
+
+    * TF_VAR_tenancy_ocid=Oracle tenanacy OCID
+    * TF_VAR_user_ocid=Oracle user OCID
+    * TF_VAR_fingerprint=Oracle API Key FingerPrint
+    * TF_VAR_private_key_path=oci_api_key.pem
+    * TF_VAR_ssh_public_key=$(cat my_oci_key.pub) 
+    * TF_VAR_region=us-ashburn-1
+    * TF_VAR_compartment_ocid=Oracle compartment OCID
+    * TF_VAR_subscription_id=Your Azure subscription ID
+    * TF_VAR_client_id=Your Azure service principal app id
+    * TF_VAR_client_secret=Your Azure service principal password
+    * TF_VAR_tenant_id=Your Azure tenant ID
+
+* From CLI, navigate to the "*azure_arc_servers_jumpstart/oci/ubuntu/terraform*"  directory of the cloned repo.    
+  
+* Export the environment variables you edited by running [*scripts/vars.sh*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/oci/ubuntu/terraform/scripts/vars.sh) with the source command as shown below. Terraform requires these to be set for the plan to execute properly. Note that this script will also be automatically executed remotely on the OCI virtual machine instance as part of the Terraform deployment. 
+* 
+    ```shell
+    source ./scripts/vars.sh
+    ```
+
+* Run the ```terraform init``` command which will download Terraform's AzureRM and OCI providers.
+
+    ![Screenshot of terraform init being run](./05.png)
+
+# Deployment
+
+* Run the ```terraform apply --auto-approve``` command and wait for the plan to finish. Upon completion, you will have an OCI virtual machine instance deployed and connected as a new Azure Arc enabled server inside a new Resource Group.
+
+* Open the Azure portal and navigate to the resource group "Arc-OCI-Demo". The virtual machine created on OCI will be visible as a resource.
+
+    ![Screenshot of Azure portal showing Azure Arc enabled server](./06.png)
+
+    ![Screenshot of OCI console showing OCI compute instance](./07.png)
+
+## Semi-Automated Deployment (Optional)
+
+As you may have noticed, the last step of the run is to register the VM as a new Azure Arc enabled server resource.
+
+![Screenshot showing azcmagent connect script](./08.png)
+
+If you want to demo/control the actual registration process, do the following:
+
+* In the [*install_arc_agent.sh.tmpl*](https://github.com/microsoft/azure_arc/blob/main/azure_arc_servers_jumpstart/oci/ubuntu/terraform/scripts/install_arc_agent.sh.tmpl) script template, comment out the "Run connect command" section and save the file.
+
+     ![Screenshot showing azcmagent connect script commented out](./09.png)
+
+* Get the public IP of the OCI instance by running ```terraform output```
```

Comment:
```suggestion
* Get the public IP of the OCI instance by running ```terraform output```
```suggestion
* Get the public IP of the OCI Virtual Machine instance by running ```terraform output```
```
```